MediaGalleryUi/view/adminhtml/web/js/image/image-actions.js

```diff
@@ -53,6 +53,16 @@ define([
             modalElement.modal('closeModal');
         },
 
+        /**
+         * Closes the image details and opens the image edit panel
+         */
+        editImageAction: function () {
+            var record = this.imageModel().getSelected().id;
+
+            this.closeModal();
```

Comment:
Why do we need to close the modal here?

Reply:
Hi @gabrieldagama, thank you for reviewing! This will close out the image view details modal before opening the edit details panel. If needed, I can sustain the view details modal then will figure out a way to trigger an update to its content after saving from the edit details panel.

Reply:
Hi @jmonteros422 I think we should do that. Currently, if you change an image title using the edit feature, after saving, the user gets back to the grid, but if he opens the Image Details again, it has the old title. So we need to update that anyway. Let me know if it isn't clear! Thanks!

Reply:
@gabrieldagama, copy that! Thank you!

Reply:
Hi @gabrieldagama, just fyi, I have took some part of issue #1575 's resolution in order to achieve the update on the Image Details after saving from the Edit Panel. Thanks!